CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Change Log
 
-## Unreleased Changes
+## 1.1.0
+
+Spotlight change:
+
+- We no longer show the `oasis1...` address for Ethereum-compatible accounts.
```

Comment:
Interesting choice :slightly_smiling_face: 

I would perhaps rather highlight:
- "Creating new wallets now generates 24-word mnemonic" since that should increase security of the wallet :slightly_smiling_face: 
- Or "Select ParaTime transactions, including Emerald deposit and withdraw, now show up in your
consensus account's transaction history" since that is an important new feature :slightly_smiling_face:

Reply:
going with the transaction history one then. lotta people on our team against using this address one as the spotlight change :sweat: